data/fixtures/scopes/python/collectionItem.unenclosed.iteration3.scope

```diff
@@ -2,10 +2,25 @@ def foo():
     global bar, baz
 ---
 
-[Range] = 1:11-1:19
+[#1 Range] =
+[#1 Domain] = 0:8-0:8
+          ><
+0| def foo():
+
+
+[#2 Range] = 1:4-1:19
+      >---------------<
+1|     global bar, baz
+
+[#2 Domain] = 1:0-1:19
+  >-------------------<
+1|     global bar, baz
```

Comment:
This domain is extra but we know from the previous code it will be eliminated when items are actually requested (`global` will be whittled away).

The intent of this type of test file is just to show the iteration search scopes, so this is proper.